use-case-monorepo-project.md

```diff
@@ -0,0 +1,272 @@
+This guide shows you how to optimize your Semaphore workflow for monorepo
+projects.
+
+## A basic monorepo project setup
+
+Let's say you have a fairly simple monorepo project that consists of:
+
+- A WEB server application located inside `/web-app/` directory
+- An iOS client application located inside `/ios/` directory
+- A separate docs web page located inside `/docs/` directory
+
+A Semaphore pipeline that is *not* optimized for a monorepo project, in this
```

Comment:
It is very easy to miss this `not` word here. 

I would avoid giving a full example of a non-optimized monorepo support. Some customers might copy this first paragraph without reading through the whole document.